app/lib/server/startup/email.js

```diff
@@ -288,6 +288,11 @@ settings.addGroup('Email', function() {
 			placeholder: 'email@domain',
 			secret: true,
 		});
+		this.add('Direct_Reply_ReplyTo_Enable', false, {
+			type: 'boolean',
+			env: true,
+			i18nLabel: 'Add_ReplyTo_Header',
```

Comment:
Following the changes on the setting's name, would you recommend changing its label too? @sampaiodiego

Reply:
oh, I didn't notice the setting name I suggested is actually very close to the `i18nLabel` :sweat_smile: 

I'd say you can remove the `i18nLabel` as the `_id` of the setting will be used if the label is missing.. :wink: 

also, I just realized it's missing an i18n key.. =)

Reply:
Ok, thanks for the suggestion! 🙂 
Is `i18nDescription` the missing key which you think that must be added here?

Reply:
no, sry :see_no_evil: 

what I meant was you should create a new i18n key at least on the English translation file (https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-i18n/i18n/en.i18n.json), if possible, add the same key to the portuguese (`pt-BR.i18n.json`) file as well.. this translation will be used as the label of setting on the Admin UI.

the key you'll add to the translation files should be the value of the `i18nDescription` property (if it exists), otherwise it can be the setting `_id`.. (I'd recommend adding a `i18nDescription` property only if the setting `_id` is not similar to the label that setting should have)..

hopefully it makes more sense now, I'm sorry the confusion.